app/gui/controller/engine-protocol/src/language_server/types.rs

```diff
@@ -651,10 +651,8 @@ pub type ExpressionId = Uuid;
 pub struct VisualisationConfiguration {
     #[allow(missing_docs)]
     pub execution_context_id: ContextId,
-    /// A qualified name of the module containing the expression which creates visualisation.
-    pub visualisation_module: String,
-    /// An enso lambda that will transform the data into expected format, i.e. `a -> a.json`.
-    pub expression:           String,
+    /// An enso function that will transform the data into expected format.
+    pub expression:           MethodPointer,
```

Comment:
If this is a function, why the name remained `expression`?

Reply:
I left the same name to keep the backward compatibility